### PR TITLE
Add nfdiv-cron-remind-applicants-apply-for-final-order job

### PR DESCRIPTION
**Pull request by danielbe11**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-1576

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by mapingo — APPROVED** [review]

File: k8s/namespaces/nfdiv/nfdiv-cron-remind-applicants-apply-for-final-order/nfdiv-cron-remind-applicants-apply-for-final-order.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-remind-applicants-apply-for-final-order
+spec:
+  releaseName: nfdiv-cron-remind-applicants-apply-for-final-order
+  chart:
+    git: git@github.com:hmcts/nfdiv-cron
+    ref: 0.0.8
```
Comment (timja) [inline k8s/namespaces/nfdiv/nfdiv-cron-remind-applicants-apply-for-final-order/nfdiv-cron-remind-applicants-apply-for-final-order.yaml:9]: https://github.com/hmcts/cnp-flux-config/pull/14258#discussion_r811853514

File: k8s/namespaces/nfdiv/nfdiv-cron-remind-applicants-apply-for-final-order/aat-00.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-remind-applicants-apply-for-final-order
+spec:
+  releaseName: nfdiv-cron-remind-applicants-apply-for-final-order
+  values:
```
Comment (timja) [inline k8s/namespaces/nfdiv/nfdiv-cron-remind-applicants-apply-for-final-order/aat-00.yaml:7]: Seems like your cronjob doesn't override env vars per environment using the environment variable?
https://github.com/hmcts/nfdiv-cron/blob/master/nfdiv-cron/values.yaml#L20-L34

**Review by timja — CHANGES_REQUESTED** [review]

File: k8s/namespaces/nfdiv/nfdiv-cron-remind-applicants-apply-for-final-order/aat-00.yaml (hunk 2 above)
Comment (linusnorton) [reply k8s/namespaces/nfdiv/nfdiv-cron-remind-applicants-apply-for-final-order/aat-00.yaml:7]: There was a reason but I haven't got the foggiest as to what it was: https://github.com/hmcts/nfdiv-cron/commit/2e43dc956d2230033c8a821cfd5affd0f64eb6a6

**Review by timja — APPROVED** [review]
I guess you can test it out and iterate

**Pull request merged**